jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/wizard/SamProjectGeneratorIntelliJShims.kt

```diff
@@ -32,6 +26,7 @@ class ValidationException : Exception()
 
 // IntelliJ shim requires a ModuleBuilder
 // UI is centralized in generator and is passed in to have access to UI elements
+// TODO: Kill this, it doesnt need to be tied to a module builder?
```

Comment:
i vaguely recall the module builder being the trigger to populate it into the IJ wizard, but maybe it was actually `ProjectTemplatesFactory`. would be nice to have the same project setup logic shared across both wizard formats, since that is a big pain point that i remember

Reply:
Ya, I want to eventually split our Generator from our Shims, but will be a further down the road problem

Reply:
if we can remove the module builder, we can get rid of the weird: take the `ValidationInfo` object and rethrow it as an exception concept as well

Reply:
ya, the shim should own does it need to be validation info or an error...

Reply:
I haven't gone down THAT rabbit hole though